asserts/snapasserts/validation_sets.go

```diff
@@ -584,3 +585,34 @@ func (v *ValidationSets) CheckPresenceInvalid(snapRef naming.SnapRef) ([]string,
 	sort.Strings(keys)
 	return keys, nil
 }
+
+// ParseValidationSet parses a validation set string (account/name or account/name=sequence)
+// and returns its individual components, or an error.
+func ParseValidationSet(arg string) (account, name string, seq int, err error) {
```

Comment:
some of the errors don't seem to be tested

Reply:
Indeed, updated.